app/utils/token-info.ts

```diff
@@ -107,7 +110,12 @@ export async function getFullTokenInfo(
     ]);
 
     if (!sdkTokenInfo) {
-        return legacyCdnTokenInfo;
+        return legacyCdnTokenInfo
+            ? {
+                  ...legacyCdnTokenInfo,
+                  verified: true,
```

Comment:
@mcintyre94, I'm presuming that _everything_ that comes from the legacy token list CDN can be presumed to be ‘verified.’ Does that make sense?